app/Models/File.php

```diff
@@ -93,6 +93,7 @@ public function getInfo()
 
         // Apparently track number can be stored with different indices as the following.
         $trackIndices = [
+            'tags.id3v2.track_number',
```

Comment:
I added this in because I did run into some songs in my personal library that weren't caught by the comments copied into tags. If you'd like, I can try and track these back down again.